challenges/es_AR/workspace_five.spi

```diff
@@ -1,16 +1,16 @@
 #------------------------------------------------------#
-# Desafío 6 : ¡Repítela!
-# Haz que repita tu música por arte de magia
+# Desaf�o 6 : �Rep�tela!
```

Comment:
It was `Desafío` but now I read `Desaf�o` ?

Reply:
Check the description here :) Somewhere somehow, Sonic Pi encodes these files to UTF-8 twice, meaning I had to switch them to Latin1 encoding...

Reply:
Oh okay

Reply:
After 3 days of digging, I couldn't find where it was coming from.. Sorry 😞